docs/pages/versions/unversioned/sdk/admob.md

```diff
@@ -48,34 +48,34 @@ import {
   AdMobBanner,
   AdMobInterstitial,
   PublisherBanner,
-  AdMobRewarded
+  AdMobRewarded,
+  setTestDeviceIDAsync,
 } from 'expo-ads-admob';
 
+// Set global test device ID
+await setTestDeviceIDAsync("EMULATOR");
```

Comment:
Would the compiler let us `await` a `Promise` in global scope in a real JS file?

Reply:
Oh, I see this does not even try to resemble a real JS file. Maybe we can leave it as it is then? 🤔

Reply:
Yeah that's right. This PR is just a minimal update that tries to fix the errors in the documentation. It's not an overhaul.